statics/workflows/check-mtu.yaml

```diff
@@ -19,10 +20,12 @@ Source: |
           var node = path[i]
           if (MTU != undefined && (node.Metadata === undefined || node.Metadata.MTU < MTU)) {
              console.log("MTU " + node.Metadata.MTU + " on node " + node.ID + " is inferior to " + MTU)
-             return false
+             checkMTU["CheckMTU"] = true
```

Comment:
any reason to change this ?

Reply:
Ohh, My mistake!

Reply:
![Screenshot from 2019-06-19 20-02-59](https://user-images.githubusercontent.com/42376860/59775798-6b718c80-92cf-11e9-8fd2-ceb1494b8528.jpg)
This looks better than this one
![Screenshot from 2019-06-19 19-51-38](https://user-images.githubusercontent.com/42376860/59775899-8f34d280-92cf-11e9-81ba-a68e3d9a0fa9.jpg)

Reply:
Ok, I will fix it on webUI side!